data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,408 @@ mission "Ringworld Debris: Quarg"
+			choice
+				`	"If the Builders were able to make self replicating spacecrafts, why aren't the Ka'het like that as well?"`
+					goto kahet
+				`	"Do they have Jump Drives on them if they were able to get here?"`
+					goto jumpdrive
+				`	"Why aren’t they in other systems?"`
+					goto jumpdrive
+			label kahet
+			`	Taely walks towards the console again. On the screen a tactical view of the Fetri'sei appears. "Well, consider that the Ka'het are true battleships, heavily shielded and always with their weapons ready. These are more akin to reconnaissance and mining probes. Their structure is extremely weak, and they do not have any weapons except the guns necessary to mine asteroids. I imagine it would take decades for even just a Maeri'het to replicate itself.`
+			`	"There is also the tactical problem: forcing a Ka'het to spend such a long time building copies of itself makes them vulnerable to enemy attacks and unable to effectively fight back, making them ill suited for the job."`
+			choice
+				`	"If they can duplicate themselves, why aren't there more Ka'sei here?"`
+					goto kasei
+				`	"Do they have Jump Drives of them if they were able to get here?"`
+				`	"Why aren’t they in other systems?"`
+			label jumpdrive
+			`	"They do not have any drive on board, not even a hyperdrive. Limited to sub-light velocity, they would need hundreds of years to pass from one system to another, which testifies how patient the Builders could be. Most species we have encountered abandon any form of sub-light transportation method between star systems as soon as hyperdrives start being developed. For all we know, the Ember Waste could have been visited by these Fetri'sei or some other Ka'sei, but they had to leave for some reason we aren't aware of. The last Builder died almost 100,000 years ago, so long that having a clear idea of what happened is impossible.`
+			choice
+				`	"What's the difference between Ka'sei and Fetri'sei?"`
+			`	"Well, there isn't," Taely responds, "not in this case at least. The naming scheme the Builders used is a bit odd: we know that 'Ka', in their language, has a meaning close to the one of 'group', 'team' and sometimes even 'ring', although it can never be used without a name next to it. So we have 'group mar', 'group het', and so on. These aren't limited to spaceships: the Ka'het, for example, includes both the biological spacecrafts around the Graveyard and the stations that constructs them, while until now we thought that the Ka'sei included only a handful of large scale construction projects. We know 'Fetri' to be a sort of spaceship sub-classification, but we aren't aware of its exact meaning. So the Fetri'sei, or 'ship sei' are a part of the Ka'sei, or 'group sei.' To put it simple, it's similar to how all squares are rectangles, but not all rectangles are squares. It just so happens that everything in this system is classified as a Fetri'sei, so we can use either interchangeably.`
```

Comment:
```suggestion
			`	"Well, there is not," Taely responds, "not in this case at least. The naming scheme the Builders used is a bit odd: we know that 'Ka', in their language, has a meaning close to the one of 'group', 'team' and sometimes even 'ring', although it can never be used without a name next to it. So we have 'group mar', 'group het', and so on. These are not limited to spaceships: the Ka'het, for example, includes both the biological spacecrafts around the Graveyard and the stations that constructs them, while until now we thought that the Ka'sei included only a handful of large scale construction projects. We know 'Fetri' to be a sort of spaceship sub-classification, but we are not aware of its exact meaning. So the Fetri'sei, or 'ship sei' are a part of the Ka'sei, or 'group sei.' To put it simply, it is similar to how all squares are rectangles, but not all rectangles are squares. It just so happens that everything in this system is classified as a Fetri'sei, so we can use either interchangeably.`
```

removing contractions from Remnant speech.